inc/commonitilobject.class.php

```diff
@@ -6896,6 +6911,11 @@ function getTimelineItems() {
          ];
       }
 
+      //if is from notif and if we want private too
+      if (!Session::haveRight("followup", ITILFollowup::SEEPRIVATE) && $params['expose_private']) {
```

Comment:
I was talking about moving this condition in the previous if.

Reply:
done